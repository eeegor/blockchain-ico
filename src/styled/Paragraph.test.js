import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Paragraph } from '.';

describe('Paragraph', () => {
	test('renders default h1', () => {
		const tree = renderer.create(<Paragraph>Title</Paragraph>).toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('renders h1 with custom color', () => {
		const tree = renderer
			.create(<Paragraph color="green">Title</Paragraph>)
			.toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('color', 'green');
	});
});
