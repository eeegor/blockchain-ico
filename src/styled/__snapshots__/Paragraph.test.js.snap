// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph renders default h1 1`] = `
.c0 {
  font-size: 14px;
  line-height: 27px;
  font-family: GTCinetype-Regular;
  margin-bottom: 12px;
}

<p
  className="c0"
>
  Title
</p>
`;

exports[`Paragraph renders h1 with custom color 1`] = `
.c0 {
  font-size: 14px;
  line-height: 27px;
  font-family: GTCinetype-Regular;
  margin-bottom: 12px;
  color: green;
}

<p
  className="c0"
  color="green"
>
  Title
</p>
`;
